### DISCUSSION TITLE
Passwordless login

### DISCUSSION THREAD

**Discussion by zizther** [craftcms/cms#9127] [Ideas] [answered]
A feature being adopted more, as well as requested more from clients is passwordless login where instead of entering a password the user only enters their email address and a token validates the authentication via an email, similar to how the verify account email Craft already sends.

Examples if this can be seen on medium.com vercel.com

Support in core would be nice to see, but possibly a plugin could fill the gap.

**Comment by andris-sevcenko** [comment, marked as answer]
I'll close this in favor of #8453 

This will be a part of Craft 4.0 core.

**Reply by andris-sevcenko** [reply]
Well, I won't close/lock it. It's not exactly the same as #8453. So I'll just say that this will be possible in Craft 4.0.

**Reply by zizther** [reply]
Amazing. Great to hear!

**Reply by zizther** [reply]
@andris-sevcenko Just to double check, this would be a passwordless login like the approach medium.com has where instead of a password field it sends the user a link to login, rather than this email being sent as part of 2FA.

**Reply by andris-sevcenko** [reply]
Yup, that's definitely going to be available. It's already possible on the feature branch, but it's super rough currently.

**Reply by zizther** [reply]
Amazing, thanks!
